src/commands/dev-env-sync-sql.ts

```diff
@@ -156,6 +156,12 @@ export class DevEnvSyncSQLCommand {
 	 * the latest backup
 	 */
 	public async generateExport(): Promise< void > {
+		if ( this.configFile ) {
+			await this.generateLiveBackupCopy();
+
+			return;
```

Comment:
Out of curiosity, why not just `return this.generateLiveBackupCopy();` and drop `await`? Minus one microtask.

Reply:
Just for simplicity, actually.

If we change the signature of `generateLiveBackupCopy`, we don't need to change anything in this function.